src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -462,7 +462,7 @@ void EnGo_InitDust(EnGoEffect effect[ENGO_OTHER_EFFECT_COUNT], Vec3f pos, Vec3f
  *
  * @param effect The EnGoEffect table
  */
-void EnGo_DrawDust(EnGoEffect effect[ENGO_EFFECT_COUNT], PlayState* play2) {
+void EnGo_DrawDust(EnGoEffect effect[], PlayState* play2) {
```

Comment:
I thought we wanted these counts in?

Reply:
I removed them before the other PR that actually fixes them, I'll add them again